framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1680,6 +1680,12 @@ public final String getOption(String name) {
     // TODO I would like to rename getLintOption to hasLintOption
     @Override
     public final boolean hasOption(String name) {
+
+        // Always ignore errors from raw type arguments.
+        // See https://github.com/typetools/checker-framework/issues/635
+        if (name.equals("ignoreRawTypeArguments")) {
```

Comment:
I don't like this way of overriding the default. In the places where we have `getOption("ignoreRawTypeArguments", false)` (hopefully only one) we should change the default to true.
That way, someone very eager can still call the checker with `-AignoreRawTypeArguments=false`